Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/inbound/PassthroughInboundAdminDistribution.java

```diff
@@ -38,31 +39,21 @@
  */
 public class PassthroughInboundAdminDistribution extends AbstractInboundAdminDistribution {
 
-    /**
-     * Constructor.
-     */
     public PassthroughInboundAdminDistribution() {
         super();
     }
 
-    /**
-     * Constructor.
-     *
-     * @param auditLogger
-     * @param adminUtils
-     * @param adapterFactory
-     */
     public PassthroughInboundAdminDistribution(AdminDistributionAuditLogger auditLogger,
-            AdminDistributionUtils adminUtils, AdapterAdminDistributionProxyObjectFactory adapterFactory) {
+        AdminDistributionUtils adminUtils, AdapterAdminDistributionProxyObjectFactory adapterFactory) {
         this.auditLogger = auditLogger;
         this.adminUtils = adminUtils;
         this.adapterFactory = adapterFactory;
     }
 
     @Override
     public void processAdminDistribution(EDXLDistribution body, AssertionType assertion) {
+        GenericDBUtils.logInfoServiceProcess(this.getClass());
```

Comment:
What is the point of all these logs here? "Flag service processing debug: PassthroughInboundAdminDistribution" would be outputted.